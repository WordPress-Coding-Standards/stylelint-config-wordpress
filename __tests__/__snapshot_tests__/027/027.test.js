'use strict';

const stylelint = require( 'stylelint' ),
	systemTestUtils = require( '../systemTestUtils.js' );

it( '027', () => {
	return stylelint.lint({
		files: [systemTestUtils.caseStylesheetGlob( '027' )],
		configFile: systemTestUtils.caseConfig( '027' ),
	}).then( ({ results }) => {
		expect( systemTestUtils.prepResults( results ) ).toMatchSnapshot();
	});
});
