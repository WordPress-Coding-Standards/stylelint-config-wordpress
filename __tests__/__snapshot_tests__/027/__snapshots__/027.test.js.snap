// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`027 1`] = `
Array [
  Object {
    "deprecations": Array [],
    "errored": true,
    "ignored": undefined,
    "invalidOptionWarnings": Array [],
    "parseErrors": Array [],
    "source": "027/stylesheet.css",
    "warnings": Array [
      Object {
        "column": 1,
        "line": 57,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 85,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 92,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 496,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 783,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 1069,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 1076,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 1186,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 1709,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 126,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 181,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 185,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 191,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 196,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 200,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 203,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 206,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 212,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 218,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 221,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 227,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 232,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 235,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 242,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 246,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 257,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 263,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 340,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 356,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 373,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 396,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 418,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 425,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 430,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 436,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 454,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 473,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 480,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 527,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 530,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 535,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 594,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 599,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 602,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 606,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 610,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 660,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 693,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 769,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 773,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 796,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 804,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 807,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 839,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 848,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 855,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 859,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 863,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 867,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 874,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 889,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 895,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 899,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 903,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 906,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 910,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 913,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 916,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 938,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 948,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 971,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 975,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 979,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 986,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 996,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1003,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1011,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1040,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1051,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1054,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1057,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1127,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1152,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1202,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1206,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1223,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1247,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1263,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1397,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1446,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1472,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1513,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1689,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1820,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1826,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1840,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1845,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1886,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 45,
        "line": 104,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 34,
        "line": 278,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 46,
        "line": 672,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 80,
        "line": 759,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 81,
        "line": 1269,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 88,
        "line": 1348,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 61,
        "line": 1568,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 30,
        "line": 1655,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 2,
        "line": 146,
        "rule": "shorthand-property-no-redundant-values",
        "severity": "error",
        "text": "Unexpected longhand value '5px 0 5px' instead of '5px 0' (shorthand-property-no-redundant-values)",
      },
      Object {
        "column": 3,
        "line": 483,
        "rule": "shorthand-property-no-redundant-values",
        "severity": "error",
        "text": "Unexpected longhand value '0 0 15px 0' instead of '0 0 15px' (shorthand-property-no-redundant-values)",
      },
      Object {
        "column": 2,
        "line": 1172,
        "rule": "shorthand-property-no-redundant-values",
        "severity": "error",
        "text": "Unexpected longhand value '5px 0 0 0' instead of '5px 0 0' (shorthand-property-no-redundant-values)",
      },
      Object {
        "column": 2,
        "line": 1411,
        "rule": "shorthand-property-no-redundant-values",
        "severity": "error",
        "text": "Unexpected longhand value '15px 0 0 0' instead of '15px 0 0' (shorthand-property-no-redundant-values)",
      },
    ],
  },
]
`;
